addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -175,6 +175,10 @@ function factory(dependencies) {
          * @returns {mail.chat_window|undefined}
          */
         _getNextVisibleUnfoldedChatWindow({ reverse = false } = {}) {
+            if (this.env.session.user_context.lang_direction === 'rtl') {
+                reverse = !reverse;
```

Comment:
Are you sure this is needed?

I might be wrong but I think `allOrderedVisible` are already ordered depending on text direction?

Reply:
Well, in RTL the tab cycling is in the wrong way.